nala/studio/acom/full-pricing-express/tests/full_pricing_express_workflow.test.js

```diff
@@ -0,0 +1,430 @@
+            await expect(card).toBeVisible();
+            await card.dblclick();
+            await expect(await editor.panel).toBeVisible();
+        });
+
+        await test.step('step-2: Edit CTA text and variant', async () => {
+            await expect(await editor.footer).toBeVisible();
+            await expect(await editor.CTA).toBeVisible();
+
+            // Edit CTA text
+            await editor.CTA.click();
+            await editor.footer.locator(editor.linkEdit).click();
+            await expect(await editor.linkText).toBeVisible();
+            await editor.linkText.fill(data.newCtaText);
+
+            // Change variant
+            if (data.newVariant) {
+                await expect(await editor.getLinkVariant(data.newVariant)).toBeVisible();
+                await (await editor.getLinkVariant(data.newVariant)).click();
+            }
+
+            // Add checkout parameters
+            if (data.checkoutParams) {
+                await expect(await editor.checkoutParameters).toBeVisible();
+                const checkoutParamsString = Object.keys(data.checkoutParams)
+                    .map((key) => `${encodeURIComponent(key)}=${encodeURIComponent(data.checkoutParams[key])}`)
+                    .join('&');
+                await editor.checkoutParameters.fill(checkoutParamsString);
+            }
+
+            await editor.linkSave.click();
+            await expect(await fullPricingExpress.cardCTA).toContainText(data.newCtaText);
+        });
+
+        await test.step('step-3: Verify CTA was updated', async () => {
+            // CTA text should be updated in the editor
+            await expect(await editor.footer).toContainText(data.newCtaText);
+            // CTA should be visible on card
+            await expect(await fullPricingExpress.cardCTA).toBeVisible();
+        });
+
+        await test.step('step-4: Save and verify', async () => {
+            await editor.saveButton.click();
+            await page.waitForTimeout(2000);
+            await expect(await editor.saveSuccess).toBeVisible();
+
+            await editor.closeEditor.click();
+            await expect(await fullPricingExpress.cardCTA).toContainText(data.newCtaText);
+
+            if (data.checkoutParams) {
+                const CTAhref = await fullPricingExpress.cardCTA.getAttribute('href');
+                let searchParams = new URLSearchParams(decodeURI(CTAhref).split('?')[1]);
+                for (const [key, value] of Object.entries(data.checkoutParams)) {
+                    expect(searchParams.get(key)).toBe(value);
+                }
+            }
+        });
+    });
+
+    // @studio-full-pricing-express-workflow-mnemonic - Parameterized mnemonic size test
+    const mnemonicSizes = ['xs', 's', 'm', 'l'];
+    for (const size of mnemonicSizes) {
+        test.skip(`Mnemonic workflow - size ${size.toUpperCase()} @studio-full-pricing-express-workflow-mnemonic-${size}`, async ({
```

Comment:
test.skip?  if test is not supposed to be ran, I suggest to remove it. It can always be generatively re-created later.